### PR TITLE
13989# Added Special Issues field definition under disabilities

### PR DESCRIPTION
**Pull request by abhijitvk**
## Description
Updates 526 All Claims schema to add specialIssues

## Testing done
- Unit testing

## Acceptance criteria
- [X] Unit tests passed

## Definition of done
- [X] Events are logged appropriately
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/config/schema.js [hunk 1]
```diff
@@ -797,6 +797,21 @@ const schema = {
...
+            type: 'array',
+            items: {
+              type: 'object',
+              required: ['code', 'name'],
+              properties: {
+                name: {
+                  type: 'string',
+                },
+                code: {
+                  type: 'string',
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/config/schema.js]: Per swagger, sounds like `code` should have `enums`? (Same for secondary.)

I think it's `['ALS', 'HEPC', 'POW', 'PTSD_1', 'PTSD_2', 'PTSD_4', 'PTSD_3']`

File: src/applications/disability-benefits/all-claims/config/schema.js [hunk 2]
```diff
@@ -797,6 +797,21 @@ const schema = {
             type: 'string',
             enum: ['NONE', 'NEW', 'SECONDARY', 'INCREASE', 'REOPEN'],
           },
+          specialIssues: {
+            type: 'array',
+            items: {
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/config/schema.js]: Not likely to make a huge difference, but should probably have an `maxItems: 100` property to match swagger

**Review by markgreenburg — COMMENTED** [review]
Looks good, think it needs some enums for the special issue code though.

File: src/applications/disability-benefits/all-claims/config/schema.js (hunk 2 above)
Comment (abhijitvk) [reply src/applications/disability-benefits/all-claims/config/schema.js]: @markgreenburg : Change done to add maxItems to match the swagger

File: src/applications/disability-benefits/all-claims/config/schema.js (hunk 1 above)
Comment (abhijitvk) [reply src/applications/disability-benefits/all-claims/config/schema.js]: @markgreenburg : enum added for the `code`

**Review by markgreenburg — APPROVED** [review]
🚀

**Pull request merged**